deploy/kubedirector/crd-app.yaml

```diff
@@ -31,21 +31,21 @@ spec:
             version:
               type: string
               minLength: 1
+            schema_version:
```

Comment:
Can/should we give this a slightly more descriptive name? I.e. what is this the schema of (otherwise it seems like it's talking about this CR format).